logexporter/cmd/main.go

```diff
@@ -59,7 +59,7 @@ var (
 
 	// Cloud provider specific logfiles.
 	awsLogs      = []string{"cloud-init-output"}
-	gceLogs      = []string{"startupscript"}
+	gceLogs      = []string{"startupscript", "konnectivity-*"}
```

Comment:
Should it be "containers/konnectivity-*" to match https://github.com/kubernetes/test-infra/commit/8fbc6048fb3d844f8c013fb34695e307fc7124ef?

Also: https://github.com/kubernetes/test-infra/commit/8fbc6048fb3d844f8c013fb34695e307fc7124ef fetches this log only on nodes (for all cloud providers) with log-dump.sh and this change fetches only for gce and both on nodes and masters.

Is the discrepancy between log-dump.sh and logexporter intended? They both try to do the same thing so I would expect them to implement similar logic.

Reply:
Working on making it better match containers. log-dump and logexporter differences were part of dims response and working on that as well. The konnectivity system is only enabled on GCE (that I know of). I intended this to be on both nodes and masters. On nodes it grabs the agent logs and on masters it should grab the server log.